packages/editor/src/components/provider/index.js

```diff
@@ -209,10 +216,13 @@ export default compose( [
 			updatePostLock,
 			createWarningNotice,
 			resetEditorBlocks,
+			resetEditorSelection,
 			updateEditorSettings,
-			resetEditorBlocksWithoutUndoLevel( blocks ) {
+			resetEditorBlocksWithoutUndoLevel( blocks, selectionStart, selectionEnd ) {
```

Comment:
These are no longer used?

```suggestion
			resetEditorBlocksWithoutUndoLevel( blocks ) {
```